api/envoy/api/v2/route/route.proto

```diff
@@ -495,23 +515,23 @@ message RouteAction {
 
   // Specifies a set of headers that will be added to requests matching this
   // route. Headers specified at this level are applied before headers from the
-  // enclosing :ref:`envoy_api_msg_route.VirtualHost` and
+  // enclosing :ref:`envoy_api_msg_route.Route` and
```

Comment:
Shouldn't this say from the enclosing Route, VirtualHost, and RouteConfiguration (with appropriate links)?